about_asserts.exs

```diff
@@ -29,10 +29,10 @@ defmodule AboutAsserts do
     is_1_equal_2? = fn -> assert 1 == 2 end
     is_1_greater_than_2? = fn -> assert 1 > 2 end
 
-    message = "Assertion with " <> __? <> " failed"
+    message = "Assertion with #{__?} failed"
     assert_raise ExUnit.AssertionError, message, is_1_equal_2?
 
-    message = "Assertion with " <> __? <> " failed"
+    message = "Assertion with #{__?} failed"
```

Comment:
I'm not sure either. I considered `Enum.join(["Assertion with", __?, "failed"], " ")`, but we haven't seen lists or `Enum` at this point in the koans.

What if we instead built the whole error message, and the blank was "which function causes this error?"